docs/theme-context.rst

```diff
@@ -0,0 +1,123 @@
+Read the Docs data passed to Sphinx build context
+=================================================
+
+Before calling `sphinx-build` to render your docs, Read the Docs injects some
+extra context in the templates by using the `html_context Sphinx setting`_ in
+the ``conf.py`` file. This extra context allows Read the Docs to add some
+features like "Edit on GitHub" and use a user custom Analytics code, among others.
+
+.. _html_context Sphinx setting: http://www.sphinx-doc.org/en/stable/config.html#confval-html_context
+
+Context injected by default
+---------------------------
+
+Here is the full list of values injected by Read the Docs as a Python
+dictionary. Note that this dictionary is injected under the main key
+`readthedocs`:
+
+
+.. code:: python
+
+    'readthedocs': {
+        'version': {
+            'pk': int,
+            'name': str,
+            'slug': str,
+            'downloads': {
+                'pdf: str,
+                'htmlzip': str,
+                'epub': str
+            },
+            'resource_uri': '/api/v2/version/{pk}/'
+        },
+        'project': {
+            'pk': int
+            'name': str,
+            'slug': str,
+            'canonical_url': str,
+            'resource_uri': '/api/v2/project/{pk}/'
+        },
+        'sphinx': {
+            'html_theme': str,
+            'source_suffix': str,
+        },
+        'analytics': {
+            'user_analytics_code': str,
+            'global_analytics_code': str
+        },
+        'vcs': {
+            'type': str,  # 'bitbucket', 'github' or 'gitlab'
+            'user': str,
+            'repo': str,
+            'commit': str,
+            'version': str,
+            'display': bool,
+            'conf_py_path': str
+        },
+        'meta': {
+            'API_HOST': str,
+            'MEDIA_URL': str, 
+            'PRODUCTION_DOMAIN': str,
+            'READTHEDOCS': True
+        }
+    }
+
+
+.. note::
+
+   By design, Read the Docs passes only static information to `sphinx-build`
+   to avoid versions to be inconsistent in case the project is updated after the version is built.
```

Comment:
Grammar issue here at:
`to avoid versions to be inconsistent`

Reply:
I don't know how to fix it :)

"keep" instead of "to be"? :/